virtumall/settings/base.py

```diff
@@ -34,9 +33,8 @@ def get_env_variable(env_key, default=None):
         return default
 
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
-
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
```

Comment:
```suggestion
BASE_DIR = Path(__file__).resolve().parents[2]
```

위와 같이 pathlib 형태가 더 나은 것 같아요,
다시 롤백한 이유가 궁금합니다. ㅎㅎ